### ISSUE TITLE
Remove docs for CREATE INVERTED INDEX syntax

### ISSUE THREAD

**Issue by rmloveland** [cockroachdb/docs#8668]
Richard Loveland (rmloveland) commented:

tl;dr: It looks like `CREATE INVERTED INDEX ...` is a CockroachDB invention that is not in PG, and thus confusing and non-portable, and we should remove it from the docs.

Longer version:

AFAICT the `CREATE INVERTED INDEX ...` syntax is a "CockroachDBism" that is not in PG (based on looking at https://www.postgresql.org/docs/9.1/sql-createindex.html and https://www.postgresql.org/docs/current/textsearch-indexes.html).  I am not able to find anything in the PG docs except `CREATE INDEX foo ON bar USING GIN/GIST(mycolumn)`.  Nor did I find any use of `CREATE INVERTED INDEX ...` in any other DBs with a cursory web search.

The fact that we support this alias and document it leads to confusion IMO.  It's also (if the above is true) non-portable.

We do support `CREATE INDEX the_idx ON the_table USING GIN(the_col)` (and GIST) now, even if we did not before.  Thus removing this alias from the docs would be a net positive IMO.

Jira Issue: DOC-804

**Comment by ericharmeling** [comment]
@awoods187 Is there any plan to remove support for `CREATE INVERTED INDEX` syntax?

**Comment by awoods187** [comment]
I think the idea is that we wouldn't use this term anymore since it's not present in Postgres. Does that make sense? cc @vy-ton
